sparse/etc/acdbdata

```diff
@@ -0,0 +1 @@
+/system/etc/acdbdata
```

Comment:
ok. edit.
also I delete redundant commits (pulseaudio-modules-droid-glue remove and rever) as mal- advise